layouts/shortcodes/cs-figure.html

```
@@ -0,0 +1,13 @@
+{{- $src := .Get "src" -}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)